Library/Formula/boost.rb

```diff
@@ -137,7 +137,6 @@ def caveats
       only supported on x86_64.
       EOS
     end
-
```

Comment:
This line is usually left here in the ` caveats `. Not sure if that's deliberate, but it's usually there :smiley_cat: